decidim-admin/app/controllers/decidim/admin/participatory_process_user_roles_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+require_dependency "decidim/admin/application_controller"
+
+module Decidim
+  module Admin
+    # Controller that allows managing all the Admins.
+    #
+    class ParticipatoryProcessUserRolesController < ApplicationController
+      def create
+        authorize! :create, Decidim::ParticipatoryProcessUserRole
+        @form = ParticipatoryProcessUserRoleForm.from_params(params)
+
+        CreateParticipatoryProcessAdmin.call(@form, participatory_process) do
+          on(:ok) do
+            flash[:notice] = I18n.t("participatory_process_user_roles.create.success", scope: "decidim.admin")
+          end
+
+          on(:invalid) do
+            flash.now[:alert] = I18n.t("participatory_process_user_roles.create.error", scope: "decidim.admin")
+          end
+          redirect_to participatory_process_path(participatory_process)
```

Comment:
1) In this PR we don't have submenus yet, so I'd go for a YAGNI approach here.
2) I can use `redirect_back`, but it still needs a `fallback_location`, which I think should be the place where the user roles are shown, which is the `participatory_process_path(participatory_process)` right now. Good old `redirect_to :back` is deprecated.